youtube/generic.md

```diff
@@ -1,50 +1,71 @@
-...
+## YouTube Channel
+Don’t forget to support my channel by subscribing it! It’s free: https://youtube.com/eddiejaoude?sub_confirmation=1
 
+MAKE SURE TO TELL YOUR FRIENDS ABOUT IT, SO THAT THEY ALSO LEARN SOMETHING NEW!
 
-## LEARN MORE
+## YouTube Shorts
+I have another YouTube channel for short clips, please TAKE A MOMENT SUBSCRIBING IT: https://youtube.com/eddiejaoudetv?sub_confirmation=1
```

Comment:
```suggestion
I have another YouTube channel for short clips, please take a moment subscribing to it: https://youtube.com/eddiejaoudetv?sub_confirmation=1
```
I think the capitals might not look very natural. (Non-violent-communication in phrasing like these might look better). Thoughts, @eddiejaoude ?